src/lib/openFile.js

```diff
@@ -70,11 +70,15 @@ export default async function openFile(file, data = {}) {
         const blob = new Blob([binData], { type: fileInfo.type });
         dialogs.box(name, `<img src='${URL.createObjectURL(blob)}'>`);
         return;
+      } else if(/video/i.test(fileInfo.type)) {
+        const blob = new Blob([binData], { type: fileInfo.type });
+        dialogs.box(name, `<video src='${URL.createObjectURL(blob)}' controls></video>`);
+        return;
       }
 
       const confirmation = await dialogs.confirm(strings.info, strings['binary file']);
       if (!confirmation) return;
-    }
```

Comment:
You need to clean this extra `else if` on line 81

Reply:
Ok done